docs/user_guide/getting_started.rst

```diff
@@ -28,6 +28,27 @@ installed. There are two options for this:
 Install boa
 ===========
 
+If you need to create a new environment:
+
+     conda create -n boa
+     conda activate boa
+
+If you don't need to create a new environment, activate that environment
+Then install dependencies
+
+    conda install botorch -c pytorch -c gpytorch -c conda-forge
+    pip install ax-platform
```

Comment:
Will this still work for Intel/rosetta Macs? Since for the dev install we have to have a different environment for those

Reply:
Maybe we need to add a test that the non-dev install works also?

Reply:
NO It doesn't! Good catch!